### PR TITLE
fix: geocoding's response examples 

### PR DESCRIPTION
**Pull request by NivGreenstein**
Hey! 
I fixed the response examples for MGRS to shape and Lat-Lon to Some Control Grid. 
Thanks!

### CODE REVIEW

File: docusaurus.config.js
Comment (shimoncohen) [inline docusaurus.config.js:1]: Please undo these changes.

File: docs/MapColonies/vector/Services/geocoding/README.md [hunk 1]
```diff
@@ -1376,20 +1385,27 @@ curl --location '<geocoding_url>/search/MGRS/tiles?tile=33UUU6099626777' \
             "match_latency_ms": 0
         }
     },
-    "bbox": [  12.948777289238832, 52.57325754975297, 12.948791616108007, 52.57326678960368 ],
+    "bbox": [ 12.948777289238832, 52.57325754975297, 12.948791616108007, 52.57326678960368],
```
Comment (shimoncohen) [inline docs/MapColonies/vector/Services/geocoding/README.md:1388]:
```suggestion
    "bbox": [ 12.948777289238832, 52.57325754975297, 12.948791616108007, 52.57326678960368 ],
```

File: docusaurus.config.js
Comment (NivGreenstein) [reply docusaurus.config.js:1]: DONE

**Review by shimoncohen — APPROVED** [review]
